src/styles/index.ts

```diff
@@ -4045,6 +4045,12 @@ const styles = (theme: ThemeColors) =>
             width: 1,
         },
 
+        userDimensionsSubtitle: {
```

Comment:
Can you show me where this is being used with a screenshot? I feel like we have existing styles for everything that's been written here.

Reply:
This is used for "View detailed instructions on adding user-defined dimensions." text
<img width="378" alt="Screenshot 2024-07-03 at 16 42 00" src="https://github.com/Expensify/App/assets/88395093/93ccff68-fd44-4e3b-bc60-1961482ca551">

Reply:
Interesting, we have a lot of other empty state pages like this - can you try to reuse styles from there? It might even give us better spacing as I think we normally have 8px space below the title. 

For instance, when you Start a chat > Room but have no workspaces, you get something very similar:
![CleanShot 2024-07-03 at 16 57 06@2x](https://github.com/Expensify/App/assets/2319350/fa75069f-bf62-4eb6-bf49-ea447000207a)

Reply:
You're right! I was able to reuse existing styles!